File: WebSocket/ws_test.js
const { type } = require('os');
const WebSocket = require('ws')
const miWebSocket = new WebSocket('ws://192.168.0.15');

let json = require('C:/Users/Valentina/Documents/1_Universidad/11S/2. Sensores/Proyecto/WebSocke/test.json');
let data = JSON.stringify(json);
function open () {
    // Abre conexión
    console.log("WebSocket abierto.");
    miWebSocket.send(data);
    miWebSocket.close();
}


function close () {
    // Cierra la conexión
    console.log("WebSocket cerrado.");
}


miWebSocket.addEventListener('open', open);
miWebSocket.addEventListener('close', close);

File: WebSocket/ws_test.py
import websocket
import json
from json import dumps

f=open('test.json')
data = json.load(f)
# Connect to WebSocket server
ws = websocket.WebSocket()
ws.connect("ws://192.168.0.15")
print("Connected to WebSocket server")

# Ask the user for some input and transmit it

text = dumps(data)
ws.send(text)
#print(str)
# Wait for server to respond and print it
result = ws.recv()
print("Received: " + result)

# Gracefully close WebSocket connection
ws.close()